requirements.txt

```
@@ -1,3 +1,3 @@
 strawberryfields>=0.15
-pennylane>=0.19
+git+git://github.com/PennyLaneAI/pennylane
```

This change will be reverted right before merging. The idea was to check that tests pass with the latest version of PennyLane. Certain was were failing ([see here](https://github.com/PennyLaneAI/plugin-test-matrix/runs/4182515243?check_suite_focus=true#step:8:46)).
```suggestion
pennylane>=0.19
```

---

Ahh makes sense